sources/mysql/infoschema.go

```diff
@@ -383,6 +383,13 @@ func toType(dataType string, columnType string, charLen sql.NullInt64, numericPr
 		return schema.Type{Name: dataType, Mods: []int64{numericPrecision.Int64, numericScale.Int64}}
 	case dataType == "decimal" && numericPrecision.Valid:
 		return schema.Type{Name: dataType, Mods: []int64{numericPrecision.Int64}}
+	case dataType == "tinyint" && len(columnType) > 7:
```

Comment:
What is the significance of `> 7`?
Seems like a magic number. Please refactor it into a constant with appropriate naming.